puppet/modules/chassis/files/chassis-hosts.service

```diff
@@ -1,13 +1,14 @@
 [Unit]
-Description=Chassis subdomain hosts updater
+Description=Avahi/mDNS CNAME publisher
```

Comment:
Is this file still needed? Can't see where it's used

Reply:
Good catch. This should be deleted now that it uses a template! Reckon you could delete it as I won't have my laptop with me for a few weeks? Taa muchly!